pie/storage/database.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import annotations
+
+from typing import Dict, Optional, Union
+
+from sqlalchemy import BigInteger, Column, String
+
+from pie.database import database, session
+
+
+class Storage(database.base):
```

Comment:
I'd call this class `StorageData` or something similar. What if we need to extend the functionality in the future?

Reply:
Fixed